graphql/schema.graphql

```diff
@@ -267,12 +288,15 @@ type CostCenter {
   paymentTerms: [PaymentTerm]
   phoneNumber: String
   businessDocument: String
+  customFields: [CustomField]
 }
 
 type B2BSetting {
   autoApprove: Boolean
   defaultPaymentTerms: [PaymentTerm]
   defaultPriceTables: [String]
+  defaultOrganizationCustomFields: [CustomField]
+  defaultCostCenterCustomFields: [CustomField]
```

Comment:
I don't think these should have the `default` prefix.

Reply:
✅